miio/vacuum.py

```diff
@@ -46,6 +79,32 @@ def __init__(self, ip: str, token: str = None, start_id: int = 0,
                  debug: int = 0) -> None:
         super().__init__(ip, token, start_id, debug)
         self.manual_seqnum = -1
+        self._model = self.info().model
+        self._init_fan_speeds()
```

Comment:
Until now the library does not do any I/O operations in constructors, so I think this should be deferred until it's really necessary. One way would be to have an internal `_fanspeedMap` function/property, which queries and stores the data during the first fetch for later uses.